common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -242,38 +243,77 @@ protected override async void DoBeginRead()
 
         protected override async void DoWrite(ChannelOutboundBuffer channelOutboundBuffer)
```

Comment:
We have "similar" implementations for both device client mqtt transport and provisioning device client mqtt transport; hence the copy-paste of the same code.